test/not_loaded_script_test.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
@Janamou this is the only tweak I'd like a reply to...then I'm happy to land this!